src/middlewared/middlewared/test/integration/assets/apps.py

```diff
@@ -0,0 +1,13 @@
+import contextlib
+
+from middlewared.test.integration.utils import call
+
+
+@contextlib.contextmanager
+def create_app(app_name: str, create_payload: dict, delete_payload: dict | None = None):
```

Comment:
We usually test assets just `app`, so that the code looks good wrt to English grammar `with app(...)`, or is there any particular reason why here it should be different?